actor/mailbox.go

```diff
@@ -201,6 +212,20 @@ func createSendHandler[T any](sendC chan<- T) sendHandler[T] {
 	}
 }
 
+func createNoopHandler[T any]() sendHandler[T] {
+	return func(_ Context, v T) error {
+		// Notify the callback.
+		if s, ok := any(v).(CallbackHook); ok {
+			// This needs to be run in a separate goroutine
+			// to avoid blocking the caller, to simulate a normal call.
+			go s.Notify(fmt.Errorf("unable to send to a stopped Mailbox"))
```

Comment:
spawning goroutine just to do `Notify` is very wasteful in this case - as it only does one operation (write to channel).  
image hypothetical scenario when there are millions of requests cumming in and all all spawning new goroutines, this would drastically impair performance of whole app. 

in this code, and in code that heavily relies on go-actor, users should never write goroutines. they should implement actors and which are themself goroutines. manually writing goroutine indicates that principle is not followed through or that there is some kind of nastiness (which sometimes can be necessary).

Reply:
Good feedback, and this was tailored to my case so performance was fine. If I can override the handler with my own func and move sync into its own library then this would go away, or at least be constrained to my use case.